### PR TITLE
 Fix account picker infinite loading during Ledger import

### PR DESCRIPTION
**Pull request by Maskln**
paired with: https://github.com/AmbireTech/ambire-app/pull/7408

## Summary
- Add `#setPageGeneration` to invalidate in-flight `setPage()` requests and clear `accountsLoading` on `reset()`
- Fix invalid page (`<= 0`) leaving the spinner active
- Apply `pageSize` from `setInitParams` earlier so the UI can become ready sooner
- Improve Ledger error normalization for benign SDK states (`none`, empty interaction)
- Add test for invalid page clearing `accountsLoading`

### CODE REVIEW

File: src/controllers/accountPicker/accountPicker.test.ts [hunk 1]
```diff
@@ -125,6 +125,24 @@ describe('AccountPicker', () => {
...
+      shouldSearchForLinkedAccounts: false,
+      shouldAddNextAccountAutomatically: false
+    })
+    await controller.init()
+    await controller.setPage({ page: 0 })
+
+    expect(controller.accountsLoading).toBe(false)
+    expect(controller.pageError).toBeTruthy()
+    expect(controller.page).toEqual(DEFAULT_PAGE)
+  })
```
Comment (superKalo) [inline src/controllers/accountPicker/accountPicker.test.ts:144]: I don't see the context behind this test. `setPage({ page: 0 })` isn't reachable anyways - every caller in the UI already guards it (pagination, navigation, etc).

This either makes no sense, or covers a corner case that's not explained?

Meanwhile the change this PR actually exists for - the `#setPageGeneration` stale-request guard - has no test coverage.

So either swap this test for one that calls two `setPage` calls (assert the later page wins), or one where `reset()` interrupts an in-flight `setPage` and test if `accountsLoading` still lands on `false`.

Those would fail before the fix and actually protect the race, which would ultimately validate that the changes in this PR are actually fixing problem(s).
